experimental/web/build

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Build browser introspection client.
+
+#######################################
+# Copies a directory without overwriting identical files at the destination.
+# Arguments:
+#   A source directory, a destination directory.
+#######################################
+function copy_directory_nondestructively() {
+  local SOURCE_DIR="$1"
+  local DESTINTAION_DIR="$2"
+
+  for file in $( (cd "${SOURCE_DIR}" && find . -type f) ); do
+    local SOURCE_FILE="${SOURCE_DIR}/${file}"
+    local DESTINATION_FILE="${DESTINTAION_DIR}/${file}"
+    if ! cmp --silent "${SOURCE_FILE}" "${DESTINATION_FILE}"; then
+      mkdir -p "$(dirname "${DESTINATION_FILE}")"
+      cp -f "${SOURCE_FILE}" "${DESTINATION_FILE}"
+    fi
+  done
+}
```

Comment:
Duplicates the function from the introspection tool, not super DRY, but given this is in `experimental` it's probs fine for now